Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h

```diff
@@ -1278,12 +1278,17 @@ class MacroAssemblerX86_64 : public AbstractMacroAssembler<Assembler> {
 
     void absDouble(FPRegisterID src, FPRegisterID dst)
     {
```

Comment:
These changes were required because of the
```
m_jit.absDouble(resultLocation.asFPR(), resultLocation.asFPR());
```
on the new `BBQJIT::addF64Copysign`

Reply:
We changed the implementation so it doesn't use the fpTempRegister anymore, WDYT?